address-space-controller/src/main/java/io/enmasse/controller/router/config/Connector.java

```diff
@@ -20,6 +20,8 @@
     private String saslMechanisms;
     private String failoverUrls;
     private Role role;
+    private Integer maxFrameSize;
```

Comment:
Do you intend to make the corresponding change to the Go types too?

Reply:
Thanks for reminding me!